build.gradle.kts

```diff
@@ -22,12 +22,19 @@ liquibase {
         )
         // Configuring luiquibase
         register("main") {
-            arguments = mapOf("changeLogFile" to "db/db.changelog-master.xml") +
+            arguments = mapOf(
+                "changeLogFile" to "db/db.changelog-master.xml",
+                "liquibaseSchemaName" to "save_cloud",
```

Comment:
why we need `liquibaseSchemaName` and `defaultSchemaName` here?

Reply:
Removed